src/editors/notepadpp.ts

```diff
@@ -0,0 +1,60 @@
+import os from "os";
+
+import Editor from "./editor";
+
+export default class Notepadpp extends Editor {
+  public static getName(): string {
+    return "Notepadpp";
```

Comment:
Let's use [Prettier](https://prettier.io/), which will reformat this to single quotes according to our [prettier config](https://github.com/wakatime/desktop/blob/master/.prettier.js).